### ISSUE TITLE
nan loss in eval

### ISSUE THREAD

**Issue by f-nozaki000** [HuayueZhang/textured_surface_anomaly_detection#1]
When I tried to run these code on DAGM class1, only first seg epoch returned conceivable values.
From seg epoch 002, all member in pred went 0.

`
print(ops['pred'].eval(feed_dict=feed_dict, session=sess).flatten())
[0. 0. 0. ... 0. 0. 0.]
`

First I doubted dying Relu, but extreamly small --base_learning_rate didn' t work.
Now I trying to clarify the way of batchnormalization in your code, confusing to me.

If you make sense what is my problem, would you give me out any advice?

**Comment by hui001** [comment]
Hello，I cant find the  DAGM dataset in the internet,  could you please tell me where I can get it? I would really appreciate it if you can just give me the dataset by email. My email is flyingupward@163.com，thank you!
